### DISCUSSION TITLE
is there one way to get the time consumption for finding one element?

### DISCUSSION THREAD

**Discussion by billychen2014** [appium/appium#18859] [Q&A]
Hello all, 
 
i'm considering testing mobile app page rendering time using appium v2  by steps below:

1. confirm one element at the target page and then trying to find it (i.e.: once the element was found , i think that the whole rendering was done)

2. cal the whole time and reduce the finding time ( the time find_element method used)

i read the corresponding document ,but i can't find the way to get it, some docs i missed? thanks

**Comment by jlipps** [comment]
I probably wouldn't use Appium to determine view render time. But if you wanted to do this, I would just start a timer when navigating to the new view, trigger an explicit wait for an element you know is only present when the view is fully rendered, and then stop it when the element is found. That should give you an upper bound on the render time.

**Reply by billychen2014** [reply]
u a right ， it's not best solution. hmm, that's  a compromise solution.
